sdk/eventhubs/azure_messaging_eventhubs/benches/benchmarks.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+use azure_identity::DefaultAzureCredential;
+use azure_messaging_eventhubs::{models::EventData, ProducerClient, SendEventOptions};
+use criterion::{criterion_group, criterion_main, Criterion};
+use std::{env, sync::Arc};
+use tokio::runtime::Runtime;
+
+static INIT_LOGGING: std::sync::Once = std::sync::Once::new();
+
+fn setup() {
+    INIT_LOGGING.call_once(|| {
+        println!("Setting up test logger...");
+
+        use tracing_subscriber::{fmt::format::FmtSpan, EnvFilter};
+        tracing_subscriber::fmt()
+            .with_env_filter(EnvFilter::from_default_env())
+            .with_span_events(FmtSpan::NEW | FmtSpan::CLOSE)
+            .with_ansi(std::env::var("NO_COLOR").map_or(true, |v| v.is_empty()))
+            .with_writer(std::io::stderr)
+            .init();
+    });
+}
```

Comment:
Probably not a good idea to enable in perf tests. I appreciate it's based on whether `RUST_LOG` is set, but do we know how much code dedicated to tracing is skipped? If it truly is as efficient as the calls you already make in the shipping crates not really doing much of anything, this is probably fine. Might want to measure, though. If anything, for perf tests, this probably isn't really necessary compared to integration tests and examples anyway, IMO.